src/components/Pane/WrappedContent.tsx

```diff
@@ -0,0 +1,29 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+export interface IWrappedContentProps extends React.Props<WrappedContent> {
+  /**
+   * Optional class name to be added to the root class
+   */
+  className?: string;
+}
+
+export class WrappedContent extends React.Component<IWrappedContentProps, any> {
```

Comment:
I was originally but saw this little trick in the Dialog control and I felt it gave more clarity for what content was what.  I could simplify with className or something else but this seemed very straightforward for the consumer.